components/OpponentSelection/OpponentSelection.js

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import Image from "next/image";
+import opponents from "../../data/opponents.json";
+
+export default function OpponentSelection({ onSelect }) {
+	const handleSelect = (selectedOpponent) => {
+		onSelect(selectedOpponent.pokemons);
+	};
+
+	return (
+		<SelectionContainer>
+			<Title>Waehle deinen Gegner</Title>
+			<OpponentList>
+				{opponents.map((opponent) => (
+					<Opponent key={opponent.id} onClick={() => handleSelect(opponent)}>
```

Comment:
Please add a `button` component inside the `li` element to use the `onClick` prop on; an `li` is not the correct choice for a click event.

Reply:
thanks for the hint Paul, I have changed. :-)